src/query/procedure/py_module.cpp

```diff
@@ -1095,7 +1097,7 @@ void CallPythonProcedure(const py::Object &py_cb, mgp_list *args, mgp_graph *gra
   std::optional<std::string> maybe_msg;
   {
     py::Object py_graph(MakePyGraph(graph, memory));
-    utils::OnScopeExit clean_up(PyObjectCleanup(py_graph, !is_batched));
```

Comment:
Why is `!is_batched` removed?

Reply:
Because PyObjectCleanup doesn't have start_gc parameter anymore since we don't run it after every query